src/custom/pages/Swap/index.tsx

```diff
@@ -375,6 +386,132 @@ const SwapButton = ({ children, showLoading, showButton = false }: SwapButtonPro
     </Text>
   )
 
+interface HighFeeContainerProps {
+  padding?: string
+  margin?: string
+  width?: string
+}
+
+// const ShowMoreButton = styled.span`
+//   text-decoration: underline;
+//   cursor: pointer;
+//   margin-left: auto;
+// `
+
+const WarningCheckboxContainer = styled.small`
+  &&&& {
```

Comment:
I'd refrain using these. I saw it used else where in the code, but this has caused issues with debugging. I'd solve the root specificity/import/override issue at hand. Akin to `!important` to me.

Reply:
_grumbling intensifies_